src/Microsoft.AspNet.Identity/IdentityErrorDescriber.cs

```diff
@@ -15,6 +15,15 @@ public virtual IdentityError DefaultError()
                 Description = Resources.DefaultError
             };
         }
+        public virtual IdentityError ConcurrencyFailure()
+        {
+            return new IdentityError
+            {
+                Code = nameof(ConcurrencyFailure),
```

Comment:
Oh I see what you are saying, you just meant as a code cleanup to use the attribute instead of nameof?  I don't particularly like that bit of magic as it doesn't even save characters in this situation